src/plugins/conditionals/README.md

```diff
@@ -103,20 +103,19 @@ sudo kdb mount main.ini /examples/conditionals ni
 sudo kdb mount sub.ini /examples/conditionals/sub ini
 
 # mount conditionals as global plugin
-sudo kdb global-mount conditionals
+sudo kdb global-mount conditionals || $(exit 0)
```

Comment:
> Did it fail for you?

Yes, I just copied this [workaround](https://github.com/ElektraInitiative/libelektra/pull/1301/files/c0b960ee6f4a92c641a0705127ee22cae79630d4#diff-d336c62f3fc46b0f169d0864fbd9c65e) from Tom’s pull request. The problem here is that calling `kdb global-mount conditionals` twice will fail (see issue #1306).